src/app/tecnologia/tecnologia.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TecnologiaComponent } from './tecnologia.component';
+
+describe('TecnologiaComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/brunodiascultural/bookshelf_v1/pullRequest?prid=8812417)
```suggestion
describe("TecnologiaComponent", () => {
```